backend/followers/views.py

```diff
@@ -0,0 +1,69 @@
+from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from rest_framework.generics import GenericAPIView
+from followers.models import Follower
+from followers.serializers import FollowerSerializer
+from authors.models import Author
+import json
+
+class FollowerView(GenericAPIView):
+    def get(self, request, aid):
+        """
+        List followers for a given author
+        """
+        #get all followers for author
+        followers = Follower.objects.filter(followed=aid)
+        serializer = FollowerSerializer(followers, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+
+    def delete(self, request, aid):
+        followers = Follower.objects.all()
+        followers.delete()
+        return HttpResponse(status=204)
+
+def createFollowerJSONPayload(request, aid, fid):
```

Comment:
This is not needed, will remove this. Was going to be used to save a list into the DB, but not sure this is this best idea

Reply:
Above comment should be about categories, not followers. Referring to wrong piece of code here.